src/webview/welcome/app/welcomePage.tsx

```diff
@@ -216,8 +227,12 @@ export class Welcome extends React.Component<DefaultProps, {
     </footer>
 
     render(): React.ReactNode {
-        const { lastRelease } = this.state;
+        const { lastRelease, isChecked } = this.state;
         return <>
+            <div className='header__logo'>
+                <FormControlLabel control={<Checkbox checked={isChecked} onChange={this.updateShowWelcomePageConfig} sx={{ color: 'white', '&.Mui-checked': { color: 'white' }}} />}
```

Comment:
When you switch to the light theme, it's hard to see the checkbox. There are a few different ways to handle this:
- you can use the css variables set by VS Code that it uses (eg. setting the colours to `var(--vscode-foreground)` or `var(--vscode-background)`), which are updated automatically when the theme changes. It's probably best to do this, since that's what's done through the rest of the UI.
- you can use [the `<ThemeProvider>` element](https://github.com/redhat-developer/vscode-openshift-tools/pull/2803/files#diff-9979b203d268885e8c70cafa96829141b87e9e033290710c79bd0b44e8f88ad3R87). The idea is that you pass a new message from VS Code to the webview to indicate that the theme has changed. Then, you update a new state variable, `theme`, with the new theme, which you can build by passing the theme kind (light or dark) to `createTheme()`. This will update the colours for all the material ui components that are a child of the `ThemeProvider` component.